Keas.Mvc/Handlers/ApiKeyMiddleware.cs

```diff
@@ -45,8 +45,7 @@ public async Task Invoke(HttpContext context, ApplicationDbContext dbContext, IO
             }
 
             // lookup team that this API key has acess to
-            var teamAccess = dbContext.Teams.FirstOrDefault(t => t.ApiCode == headerGuidValue);
-
+            var teamAccess = dbContext.Teams.Include(a => a.TeamApiCode).FirstOrDefault(a => a.TeamApiCode != null && a.TeamApiCode.ApiCode == headerGuidValue);
```

Comment:
You don't need the .Include because you aren't using it in the return. Also I don't think you need the null check since it's ignored in the generated sql but I don't think it hurts anything.